python/lsst/rubintv/production/allSky.py

```diff
@@ -376,13 +476,14 @@ def convertFiles(self, files, forceRegen=False):
         for file in sorted(files):  # sort just helps debug
             outputFilename = self._getConvertedFilename(file)
             self.log.debug(f"Converting {file} to {outputFilename}")
+            date, time = getDateTimeFromExif(file, logger=self.log)
```

Comment:
Won't this will raise a `TypeError` if the `DateTime` entry is missing from the EXIF data? It will return `None`, which my quick test tells me "is not iterable."

Reply:
I don't ever expect them to be missing, but this is a good point. I've changed it to return two empty strings so that the interface is always the same, and to only pass them through for adding if they contain text, so that should sort that out.